### PR TITLE
Styling and adding an image  

### PR DESCRIPTION
**Pull request by frannyfra**

### CODE REVIEW

**Review by victormasson21 — COMMENTED** [review]
Lookin groovy

**Review by victormasson21 — APPROVED** [review]

File: index.html [hunk 1]
```diff
@@ -20,11 +21,13 @@
 
       </br>
 
-  <section class="greenSection">
+  <section class="blueSection">
 
-  <h1 class="bigTitle" id="home">ABCD-group</h1>
-  <p>A B C (D), easy as one, two, three
-      Are simple as do re mi</p>
+    <img src="jacksons.png" ></img>
```
Comment (sarahyjja) [inline index.html:26]: Please, could you add an <alt> attribute?

File: style.css [hunk 2]
```diff
@@ -1,43 +1,58 @@
...
+  position: fixed;
+  top: 0;
+  width: 100%;
+}
     
-    li {
-    float: left;
-    }
+li {
+  float: left;
```
Comment (sarahyjja) [inline style.css:30]: What about working with flexbox?

**Review by sarahyjja — COMMENTED** [review]
A couple micro things to adapt for the accessibility and maybe using flexbox rathan float to make it more flexible? Just a thought anyway :)

**Review by emaggy — APPROVED** [review]
Issue with the ul style in style.css file. Been used for the nav bar but conflicts with the ul used within the main html text

**Review by sarahyjja — APPROVED** [review]

**Pull request merged**